src/utilities/getTokenIdFromVAddress.ts

```diff
@@ -0,0 +1,14 @@
+import vBepTokens from 'constants/contracts/addresses/vBepTokens.json';
+import { TokenId } from 'types';
+
+const getTokenFromVAddress = (address: string) => {
```

Comment:
You are returning the vToken ID so I'd rename the function accordingly:
```suggestion
const getVTokenIdFromVAddress = (address: string) => {
```

Reply:
This is not returning the vTokenID. It is returning the tokenId. e.g. `bnb`. 
[Source](https://github.com/VenusProtocol/venus-protocol-interface/blob/develop/src/constants/contracts/addresses/vBepTokens.json)